src/components/backend-ai-session-list.ts

```diff
@@ -4318,15 +4284,12 @@ ${rowData.item[this.sessionNameField]}</pre
         <vaadin-grid id="list-grid" theme="row-stripes column-borders compact dark" aria-label="Session list"
           .items="${this.compute_sessions}" height-by-rows>
           ${
-            this._isRunning
+            this._isRunning || this._APIMajorVersion > 4
```

Comment:
```suggestion
            this._isRunning
```
This code adds a selection column to the terminated session tabs. Let's remove it.